style.css

```diff
@@ -24,6 +24,60 @@ nav {
   position: relative;
 }
 
+.popup-card {
+  display: none;
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  overflow: auto;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  background-color: rgba(0, 0, 0, 0.9);
+}
+
+.modal-body {
+  background-color: #fefefe;
+  margin: 5% auto;
+  padding: 1.25rem;
+  border: 1px solid #888;
+  width: 80%;
+  border-radius: 8px;
+}
```

Comment:
- [ ] Kindly make sure to use the correct color around the modal. Currently, the color you are using is dark while the template uses `#C1C7D0`. Let's try adhering to the template as much as possible. 😎️

**Your design**
![Screenshot from 2023-06-21 09-13-40](https://github.com/NewtonOmbese/Newton-Portfolio/assets/17243325/f1789a20-c7b7-43d5-abb5-10611ec2bad5)

**Figma design**
![Screenshot from 2023-06-21 09-13-25](https://github.com/NewtonOmbese/Newton-Portfolio/assets/17243325/95ae3f2e-5825-46da-8c61-f7a8854c7e26)